scripts/static-libraries/build-and-push-static-libs.sh

```diff
@@ -36,11 +36,11 @@ aws s3 cp out/static-consensus-$VERSION_TAG.tar.gz s3://static-libraries.concord
 aws s3 cp out/static-consensus-binaries-$VERSION_TAG.tar.gz s3://static-libraries.concordium.com/ --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
 
 # Commit and push changes
-rm -rf out
-echo "Going to push to branch $GIT_BRANCH"
-echo $VERSION_TAG > scripts/static-libraries/LATEST_STATIC_LIBRARIES
-git add scripts/static-libraries/LATEST_STATIC_LIBRARIES
+# rm -rf out
+# echo "Going to push to branch $GIT_BRANCH"
+# echo $VERSION_TAG > scripts/static-libraries/LATEST_STATIC_LIBRARIES
+# git add scripts/static-libraries/LATEST_STATIC_LIBRARIES
 # CI is skipped by default for these commits as the CI doesn't use the
 # static libraries so the outcome would be the same as in the previous commit.
-git commit -m "Jenkins: push $VERSION_TAG static libraries. [skip ci]"
-git push origin HEAD:$BRANCH
+# git commit -m "Jenkins: push $VERSION_TAG static libraries. [skip ci]"
+# git push origin HEAD:$BRANCH
```

Comment:
Delete?

Reply:
Or better, replace with an `echo` of what should be done. Ideally a one-liner to be copy/pasted/run.

Reply:
Done.